tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -103,7 +125,11 @@ expect(() => {}).toTriggerReadyStateChanges(
   ]
 );
 
-[1 /* first comment */, 2 /* second comment */, 3];
+[
+  1 /* first comment */,
+  2, // second comment
```

Comment:
This one is actually incorrect, but there's another bug in the flow parser where it reports an incorrect location for the comment.

![image](https://cloud.githubusercontent.com/assets/197597/23149132/ca0e19e4-f79e-11e6-842d-18aa1e123d42.png)